src/main/java/com/gamedoora/backend/userservices/api/UserServicesController.java

```diff
@@ -36,21 +36,21 @@ public ResponseEntity<UserDTO> createUsers(@RequestBody UserDTO usersDto) {
   }
 
   @PutMapping(
-      value = "/{id}",
+      value = "/{emailId}",
```

Comment:
Email Id can have characters that can interfere with path. QueryParam would be a better option.